packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFailedPaymentsCancellation/index.mjml

```diff
@@ -0,0 +1,19 @@
+<mj-section>
```

Comment:
The other mjml files have a source code form comment, can you please add that in?